src/Microsoft.DotNet.Interactive.NetStandard20/Microsoft.DotNet.Interactive.Netstandard20.csproj

```diff
@@ -11,6 +11,8 @@
 
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)../Microsoft.DotNet.Interactive/**/*.cs" />
+    <Compile Remove="$(MSBuildThisFileDirectory)../Microsoft.DotNet.Interactive/bin/**/*.cs" />
```

Comment:
Why is this necessary, shouldn't generated files be included?

Reply:
@brettfo found that they get included twice, running a dotnet build is causing that issue

Reply:
In Diego's environment, the auto-generated files are under `bin/` and `obj/` and the netstandard2.0 project should certainly include its own generated files, but not those from the non-netstandard2.0 project.